src/js/_store/modules/categories.js

```diff
@@ -35,9 +39,17 @@ export default {
     clearTargetInput(state) {
       state.targetCategory = '';
     },
+    confirmDeleteCategory(state, confirmCategory) {
+      state.deleteCategory.id = confirmCategory.id;
+      state.deleteCategory.name = confirmCategory.name;
```

Comment:
[imo]
まとめて代入しても良いかなと思いました！

```suggestion
      state.deleteCategory = confirmCategory;
```

Reply:
こちら修正いたしました。